Makefile

```diff
@@ -99,6 +106,7 @@ clean: clean-integration
 	@echo "🧹 ... 🗑️"
 	@rm -rf $(OUTDIR)
 	@rm -rf $(CURDIR)/release/
+	@rm -rf $(COVDIR)
```

Comment:
This fails when running integration tests because the binary runs with elevated permissions. Is there a way to fix this?

Reply:
The _way_ would be to map the user running integration tests to the root user in the container for integration tests, but that might have its own set of challenges.

Reply:
Oh, I forgot to resolve this, the reason we had this issue was that the folder itself was being created by root instead of by the user. I think somewhere in the testing suite setup or when setting up the containers it saw the folder wasn't created and did it on its own. My newest change just adds the `mkdir` comment.